papyri/crosslink.py

```diff
@@ -12,11 +12,11 @@
 import cbor2
 from there import print
 
+import papyri.common_ast
```

Comment:
Try to get used to follow the style of the project you are in and not mix relative and pull import. Both are fine and have pros and cons, but you usually want to stick to the same style. 

Here it uses `from .XXXX` so you want to do `from . import common_ast`. 

Relative imports make it (a bit) easier to detect import cycles as you often to have `from ..` as you import a parent, and make it easier to not depend on your current package name.

Reply:
Totally agree, I had not done any cleaning on this PR, as of yet. These imports are auto generated by my IDE's one click import functionality.

Reply:
I usually do cleaning, when the general architecture/logic is approved.